src/msw/control.cpp

```diff
@@ -156,6 +156,12 @@ bool wxControl::MSWCreateControl(const wxChar *classname,
         return false;
     }
 
+    // fix position if limited by Short range
```

Comment:
There are other occurrences of `CreateWindowEx()` in wxMSW code, e.g. for creating the buddy window of wxSpinCtrl. We probably need a wrapper for it taking care of moving the window to the correct location.